public/js/util/source-map.js

```diff
@@ -71,33 +62,25 @@ function createOriginalSources(generatedSource) {
     }));
 }
 
-function getOriginalSource(originalSource, getState, dispatch, loadSourceText) {
-  return Task.spawn(function* () {
-    const generatedSource = getSource(
-      getState(),
-      getGeneratedSourceId(originalSource)
-    );
-
-    const generatedSourcetext = yield dispatch(
-      loadSourceText(generatedSource.toJS())
-    );
-
-    const originalSourceContent = getOriginalSourceContent(
-      originalSource,
-      generatedSource,
-      generatedSourcetext.text
-    );
-
-    return {
-      source: originalSourceContent,
-      contentType: "text/javascript"
-    };
-  });
+function getOriginalSource(
+  originalSource, generatedSource, generatedSourceText) {
```

Comment:
nit: this alignment looks a little weird because the args are indented with the code, and we (devtools) usually do:

```
function foo(
  x, y, z
) {
```